tests/cli/test_rasa_test.py

```diff
@@ -107,6 +107,35 @@ def test_test_nlu_cross_validation(run_in_simple_project: Callable[..., RunResul
     assert os.path.exists("results/intent_confusion_matrix.png")
 
 
+def test_test_nlu_cross_validation_with_autoconfig(
+    testdir: Testdir, moodbot_nlu_data_path: Path
+):
+    os.environ["LOG_LEVEL"] = "ERROR"
+    config_path = str(testdir.tmpdir / "config.yml")
+    nlu_path = str(testdir.tmpdir / "nlu.yml")
+    shutil.copy(str(moodbot_nlu_data_path), nlu_path)
+    write_yaml(
+        {"language": "en", "pipeline": [], "policies": [],}, config_path,
+    )
+    args = [
+        shutil.which("rasa"),
+        "test",
+        "nlu",
+        "--cross-validation",
+        "-c",
+        "config.yml",
+        "--nlu",
+        "nlu.yml",
+    ]
+
+    # we don't wanna run the cross validation for real, just want to see that it does
+    # not crash
+    try:
+        testdir.run(*args, timeout=10.0)
```

Comment:
Does this mean the test will take 10 seconds? And if so can we make it a bit quicker?

Reply:
I reduced it to 8 seconds. On my machine anything less than 5 seconds was too little to catch the problem - Unfortunately, it invokes the rasa cli which takes a long time to start up. The solution I had earlier was able to do it in 3. That just had the small side effect that we used that autoconfig system directly, which also wrote back the default config to the designated config file. But there I also didn't have to change the signature of the cross eval method. So idk what's better. Almost see the first solution as slightly better. What do you think?

Reply:
I think this way is safer as it keeps the auto config stuff encapsulated.